projects/shepherd/src/lib/shepherd.service.ts

```diff
@@ -81,6 +81,14 @@ export class ShepherdService {
     this.isActive = false;
   }
 
+  /**
+   * This function is called to remove steps from the tour.
+   * @param name is the name of the step you are looking to remove
+   */ 
+  removeStep(name: string) {
+    this.tourObject.removeStep(name);
```

Comment:
I'm curious, what is the benefit to exposing this rather than just calling `this.tourObject.removeStep` directly?

Reply:
When you are implementing logic regularly to interact with this part of shepherd.js,  you would need to import it to the component which increases code replication. The implementation I am doing has multiple tours and a majority of tours could be improved by displaying only relevant steps on the tour.

Reply:
@MinnesotaJames I still don't understand. Why not do `this.shepherdService.tourObject.removeStep` directly in your Angular app?

Reply:
You make valid, must have been identifying the objects wrong in my PoC that I was using to reference.